plz-event-source.el

```diff
@@ -9,7 +9,11 @@
 ;; Package-Requires: ((emacs "26.3"))
 ;; Keywords: comm, network, http
 
-;; This file is part of GNU Emacs.
```

Comment:
Please leave this in, I think this part of standard boilerplate - regardless of the temporary state, it really is part of GNU Emacs (which is why you needed the FSF copyright assignment)

Reply:
Yes, true. I restored it.